configs/grinn_chiliboard_serial_defconfig

```diff
@@ -0,0 +1,38 @@
+BR2_arm=y
+BR2_cortex_a8=y
+BR2_TARGET_GENERIC_GETTY_PORT="ttyO0"
+BR2_SYSTEM_DHCP="eth0"
+BR2_ROOTFS_OVERLAY="$(BR2_EXTERNAL_GRINN_EXAMPLES_PATH)/board/grinn/chiliboard-serial/rootfs-overlay/"
```

Comment:
We are not consistent with that directory names for different configurations. Please unify that.